src/users/users.service.ts

```diff
@@ -1,14 +1,43 @@
-import { Injectable } from '@nestjs/common';
+import { Injectable, Logger } from '@nestjs/common';
 import { UserProfile as Users } from './users.interface';
```

Comment:
Use DTOs as well look in @nestjs/mapped-types

Reply:
Dai I have used DTOs and entity in upcoming PR where I have setup prisma and postgres.